### PR TITLE
Feature/ddbteam 802 upgrade and maintenance

### PR DESCRIPTION
**Pull request by turegjorup**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "license": "UNLICENSED",
```
Comment (cableman) [inline package.json:2]: MIT?

File: .markdownlint.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "default": true,
+  "line-length": false
+}
```
Comment (cableman) [inline .markdownlint.json:4]: New line

File: package.json (hunk 1 above)
Comment (cableman) [reply package.json:2]: Think there should be an name and description in here too: https://docs.npmjs.com/cli/v7/configuring-npm/package-json

Comment (cableman) [reply package.json:2]: When is this nodejs packages used? I see no nodejs in `docker-compose.yml` or in actions?

**Review by cableman — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (turegjorup) [reply package.json:2]: See the readme about markdown lint

Comment (turegjorup) [reply package.json:2]: This is tooling setup, not part of the application, so I consider this to accurate
```json
  "license": "UNLICENSED",
  "private": true,
```

**Review by cableman — APPROVED** [review]

**Pull request merged**